src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,86 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
+long-summary: calculate price for reservation
+parameters:
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscpe
+  - name: --applied-scopes
+    type: string
+    short-summary: Subscription that the benefit will be applied. Do not specify if AppliedScopeType is Shared.
+  - name: --billing-plan
+    type: string
+    short-summary: choose Mothly or Upfront
+  - name: --scope
+    type: string
+    short-summary: subscription that will be charged
+  - name: --display-name
+    type: string
+    short-summary: custom name
+  - name: --quantity
+    type: int(32)
+    short-summary: quantity of RIs
+  - name: --renew
+    type: bool
+    short-summary: setup auto renewal
+  - name: --rrp
+    type: string
+    short-summary: Type of the Instance Flexibility to update the reservation with
+  - name: --reserved-resource-type
+    type: string
+    short-summary: type of RIs
+  - name: --sku
+    type: string
+    short-summary: sku name
+  - name: --term
+    type: string
+    short-summary: P1Y or P3Y choose 1 year or 3 years RI
+"""
+
+helps['reservations reservation-order purchase'] = """
+type: command
+short-summary: purchase
+long-summary: purchase a new reservation
+parameters:
+  - name: --reservation-order-id
+    type: string
+    short-summary: Id of reservation order to purchase
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscpe
+  - name: --applied-scopes
+    type: string
+    short-summary: Subscription that the benefit will be applied. Do not specify if AppliedScopeType is Shared.
+  - name: --billing-plan
+    type: string
+    short-summary: choose Mothly or Upfront
+  - name: --scope
+    type: string
+    short-summary: subscription that will be charged
+  - name: --display-name
+    type: string
+    short-summary: custom name
+  - name: --quantity
+    type: int(32)
+    short-summary: quantity of RIs
+  - name: --renew
+    type: bool
```

Comment:
`bool` should be defined as `arg_type=get_three_state_flag()`, such as https://github.com/Azure/azure-cli/blob/4d5b1a942e48ce6f8674420f2b18d50f19ed860e/src/azure-cli/azure/cli/command_modules/role/_params.py#L81

doc: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags

Reply:
modified